infrastructure/bls-keystore/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
```

Comment:
Will this be built as jar? We need to be able to pull this in as a dependency in web3signer

Reply:
in our installdist we currently have:
```
teku-infrastructure-bls-keystore-develop.jar
```
so yes, it should be able to be used... let me know if thats all you need?

Reply:
Yeah that all I need